meeting_guide/wagtail_hooks.py

```diff
@@ -69,44 +67,45 @@ class MeetingTypeAdmin(ModelAdmin):
         "display_order",
     )
     ordering = ("display_order", "type_name")
+    search_fields = ("type_name",)

```

Comment:
After `python manage.py loaddata spec_meeting_types.json` there were a lot of Meeting Types to go through. This simply searches the `type_name` when managing all MeetingTypes in the admin.